src/stylesheets/theme/_sds-theme-custom-styles.scss

```diff
@@ -42,6 +42,7 @@ SDS THEME CUSTOM STYLES
 // Layouts
 // -------------------------------------
 @import '../layouts/subheader';
```

Comment:
I'll be removing `subheader` as this seems to be causing confusion about the structure of the css